tons of error message for refresh status of execution after upgrade

2.0.4 -> 2.2.0 using helm chart

There were some scan/replication/retention/GC executed

After the upgrade I see there're tons for error message like this:

```
2021-01-25T09:52:37Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:37Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
2021-01-25T09:52:38Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"UNKNOWN","message":"unknown: failed to refresh the status of the execution 1 after 5 retries"}]}
```

--- a/make/migrations/postgresql/0050_2.2.0_schema.up.sql
+++ b/make/migrations/postgresql/0050_2.2.0_schema.up.sql
@@ -519,6 +519,9 @@ CREATE TABLE IF NOT EXISTS "report_vulnerability_record" (
     CONSTRAINT fk_report_uuid FOREIGN  KEY(report_uuid) REFERENCES scan_report(uuid) ON DELETE CASCADE
 );
 
+/*make sure the revision of execution isn't null*/
+UPDATE execution SET revision=0 WHERE revision IS NULL;
+
 /*delete the retention execution records whose policy doesn't exist*/
 DELETE FROM retention_execution
 WHERE id IN (SELECT re.id FROM retention_execution re

